packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/ios/enrollmentintegrationsteps.md

```diff
@@ -0,0 +1,523 @@
+>**Note:** The examples in this guide use Swift 5, SwiftUI, and a target of iOS 15 or later. The SwiftUI portions pass state information using multiple bindings, consider using more robust ways of sharing and updating data in production apps. For more information, see the [Apple developer documentation for state and data flow in SwiftUI](https://developer.apple.com/documentation/swiftui/state-and-data-flow/).
+
+The code in this guide manages the sign-on flow using a singleton class (`SignInController`). The class handles asynchronous responses from the SDK by implementing the delegate functions of the `IDXClientDelegate` protocol. The SDK also provides completion handlers and Swift Concurrency (`async/await`).
+
+### 1: Display the initial sign-in UI
+
+Enable the user to start the sign-in flow, such as presenting a Sign In button or a form with fields for a username and password (you won't use those values until step 3 when you [authenticate the user credentials](#_3-authenticate-the-user-credentials)).
+
+### 2: Start the sign-on flow
+
+Initialize the SDK client and request the initial server response after the user starts the sign-in process.
+
+This code also sets the caller as the delegate for asynchronous callbacks from the SDK:
+
+```swift
+// Initialize the SDK client using a Configuration object loaded previously.
+IDXClient.start(with: configuration) { (client, error) in
+    guard let client = client else {
+		self.finish(with: error)
+		return
+	}
+
+	self.client = client
+
+    // Set this object as the delegate for asynchronous calls.
+	client.delegate = self
+
+	// Request the first response from the Okta org.
+	client.resume()
+}
+```
+
+The `configuration` (`IDXClient.IDXClientConfiguration`) parameter in the call to `IDX.start(with:)` contains the configuration information for your Okta org, and is usually initialized from an `Okta.plist` file. The call to `client.resume()` requests the first response of the sign-sequence.
```

Comment:
I updated the text with a slight modification based on the Apple convention for denoting an API: `IDX.start(with:)` instead of `IDX.start()`... the latter is a call that takes no arguments.

Reply:
np. I was unaware of the apple convention for denoting an API.